modules/consensus/accept_test.go

```diff
@@ -1811,12 +1305,5 @@ func TestTaxHardfork(t *testing.T) {
 			t.Fatal(err)
 		}
 	}
-
-	// Run a siacoins check to make sure that order has been restored - note
-	// that the siacoin check will fail in the middle, and thus is commented
-	// out until after the hardfork.
-	err = cst.cs.checkSiacoins()
-	if err != nil {
-		t.Fatal(err)
-	}
 }
+*/
```

Comment:
It will eventually be restored in some capacity, probably in the form of a new test that does the same thing but without using deprecated paradigms